docs/guides/references/changelog.mdx

```diff
@@ -6,6 +6,17 @@ title: Changelog
 
 _Released 08/29/2023_
 
+**Summary:**
+
+Test data for debugging in Cypress Cloud using [Test Replay](/guides/cloud/debugging/test-replay) is enabled by default, with an opt-out available in Cloud [project settings](/guides/cloud/debugging/test-replay#Opt-out-of-Test-Replay).
+
+Test Replay now serves as the primary replacement for debugging via video. Video capture and `videoCompression` of captured video are now set to `false` by default and `videoUploadOnPasses` is removed.
+
+Additionally in this release, updates were made to the retry-ability of `cy.readFile()`. Version support for Node and Typrscript are updated and some corrections were made to the Cypress Module API properties.
```

Comment:
@jaffrepaul I dont think I’d include this in the summary. They’re just asides that don’t require context that they’ll read below in the changelog. 
```suggestion
```